tests/mock_jsons/0.5.4/create_config_map.json

```diff
@@ -2,7 +2,7 @@
     "apiVersion": "v1",
     "data": {
         "special.how": "very",
-        "special.type": "charm"
+        "special.type": {"quark": "charm"}
```

Comment:
This doesn't look right.

The *actual ConfigMap* object that OpenShift stories is key:val (str:str).

Our create_config_map(name, data) API stores data objects in that str value by JSON encoding them.

So I would expect this to look like:
```
"special.type": "{\"quark\": \"charm\"}"
```